website/views/layouts/layout.ejs

```diff
@@ -55,74 +55,76 @@
   </head>
   <body>
     <div purpose="page-wrap">
-      <div style="max-width: 1248px; height: 94px;" class="<%= isHomepage ? 'homepage-header' : 'header' %> container-fluid d-flex justify-content-between align-items-center pt-3 pb-3 px-3 px-md-4" purpose="page-header">
-        <a href="/">
-          <% /* The homepage-header on the homepage has a white fleet logo */ %>
-          <% if (isHomepage) { %>
-          <img alt="Fleet logo" src="/images/logo-white-118x48@2x.png" style="width: 118px; height: 48px;"/>
-          <% } else { %>
-          <img alt="Fleet logo" src="/images/logo-blue-162x92@2x.png" style="height: 92px; width: 162px;"/>
-          <% } %>
-        </a>
-        <div class="d-flex d-lg-none">
-          <button style="font-size: 16px; text-decoration: none;" class="header-btn btn btn-link d-flex align-items-center" data-toggle="collapse" data-target="#navbarToggleExternalContent">
-            <span class="mr-2">Menu</span>
-            <% /* The hamburger icon on the homepage is white */ %>
+      <div class="<%= isHomepage ? 'homepage-header' : 'header' %>" purpose="page-header">
+        <div style="max-width: 1248px; height: 94px;" class="container-fluid d-flex justify-content-between align-items-center pt-3 pb-3 px-3 px-md-4">
+          <a href="/">
+            <% /* The homepage-header on the homepage has a white fleet logo */ %>
             <% if (isHomepage) { %>
-            <img alt="An icon indicating that interacting with this button will open the navigation menu." src="/images/icon-hamburger-16x14@2x.png" style="width: 16px;" />
+            <img alt="Fleet logo" src="/images/logo-white-118x48@2x.png" style="width: 118px; height: 48px;"/>
             <% } else { %>
-            <img alt="An icon indicating that interacting with this button will open the navigation menu." src="/images/icon-hamburger-blue-16x14@2x.png" style="width: 16px;" />
+            <img alt="Fleet logo" src="/images/logo-blue-162x92@2x.png" style="height: 92px; width: 162px;"/>
```

Comment:
The additional padding on the blue logo is causing the "Get started" button to break into two lines at 991px - 997px width. It looks like this can be fixed by setting a max-width on the <a> tag

Reply:
@eashaw, would you mind dropping in the max-width? I'm not sure exactly where you are referring to.